opm/parser/eclipse/Deck/DeckKeyword.hpp

```diff
@@ -25,18 +25,19 @@
 #include <memory>
 #include <utility>
 
+#include <opm/parser/eclipse/Parser/ParserKeyword.hpp>
 #include <opm/parser/eclipse/Deck/DeckRecord.hpp>
 
 namespace Opm {
-    class ParserKeyword;
     class DeckOutput;
 
     class DeckKeyword {
     public:
         typedef std::vector< DeckRecord >::const_iterator const_iterator;
 
-        explicit DeckKeyword(const std::string& keywordName);
-        DeckKeyword(const std::string& keywordName, bool knownKeyword);
+        DeckKeyword(const ParserKeyword * parserKeyword);
```

Comment:
I would suggest to make these two constructors explicit, assuming you do not depend on implicit conversion.

Reply:
Done